CppCustomVisualizer/dll/_EntryPoint.cpp

```diff
@@ -0,0 +1,371 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+// _CppCustomVisualizerService.cpp : Implementation of CCppCustomVisualizerService
+
+#include "stdafx.h"
+#include "_EntryPoint.h"
+
+HRESULT STDMETHODCALLTYPE CCppCustomVisualizerService::EvaluateVisualizedExpression(
+    _In_ Evaluation::DkmVisualizedExpression* pVisualizedExpression,
+    _Deref_out_opt_ Evaluation::DkmEvaluationResult** ppResultObject
+    )
+{
+    HRESULT hr;
+
+    // This method is called to visualize the FILETIME variable. Its basic job is to create
```

Comment:
>the FILETIME variable [](start = 42, length = 21)

Should be "FILETIME variables"

Reply:
Or better 'a FILETIME variable'

---
In reply to: [251160417](https://github.com/Microsoft/ConcordExtensibilitySamples/pull/42#discussion_r251160417) [](ancestors = 251160417)